scripts/us_cdc/environmental_health_toxicology/OzoneCensusTractPollution.tmcf

```diff
@@ -0,0 +1,36 @@
+Node: E:OzoneCTPollution->E1
+typeOf: dcs:CensusTract
+dcid: C:OzoneCTPollution->dcid
+
+Node: E:OzoneCTPollution->E2
+observationAbout: E:OzoneCTPollution->E1
```

Comment:
This can just be:

observationAbout: C:OzoneCTPollution->dcid

and you can get rid of E1.

Reply:
ditto for other TMCFs too